addon/models/file.js

```diff
@@ -42,5 +42,6 @@ export default OsfModel.extend(FileItemMixin, {
     // File attributes
     versions: DS.hasMany('file-version'),
     comments: DS.hasMany('comment'),
+    node: DS.belongsTo('node'),  // TODO: In the future apiv2 may also need to support this pointing at nodes OR registrations
```

Comment:
Haven't delved into this item too much as yet; held off for now to limit scope creep.